R/PipeOpFeatureUnion.R

```diff
@@ -11,23 +11,27 @@ PipeOpFeatureUnion = R6Class("PipeOpFeatureUnion",
     train = function(inputs) {
 
       ## check if all target_names are equal
-      is_target_equal <- length(unique(vapply(
+      is_target_equal = length(unique(vapply(
         inputs,
         function(x) digest::digest(x$target_names),
         FUN.VALUE = ""
       ))) == 1
 
-      all_data <- lapply(inputs, function(x) {
+      all_data = lapply(inputs, function(x) {
         x$data(cols = x$feature_names)
       })
 
-      input1 <- inputs[[1]]
-      targets <- input1$data(cols = task$target_names)
+      input1 = inputs[[1]]
+      targets = input1$data(cols = input1$target_names)
 
-      data <- do.call(cbind, c(all_data, list(targets)))
-      db <- as_data_backend(data)
+      data = do.call(cbind, c(all_data, list(targets)))
+      db = as_data_backend(data)
```

Comment:
Michel proposed new methods in (https://github.com/mlr-org/mlr3/issues/61)
maybe we can use them here?
```
task$select(task$target_names)
task$cbind(newdata)
```

Reply:
I think we can change that when we will have the whole flow of tasks described.

Right now creating new task is safer, because if the same task will be passed to two nodes we won't break anything.

And changing the behavior here was not the purpose of this pull request. I just replaced the `<-` with `=`:)